python/tvm/relay/analysis/analysis.py

```diff
@@ -319,6 +319,22 @@ def get_total_mac_number(expr):
     return _ffi_api.GetTotalMacNumber(expr)
 
 
+def is_compute_heavy_graph(expr):
```

Comment:
I think it might be better to have this implemented in python and localized to tensorrt.py script. I don't know if this visitor is general enough to be merged to upstream like this

Reply:
Agreed. Lets try to keep the changes local to tensorrt.py. Anything outside needs to go to Apache TVM

Reply:
I see, okay , I will make the changes

Reply:
Done